packages/frontend/src/components/admin/tabs/OffersTab.tsx

```
@@ -0,0 +1,115 @@
+  onEdit: (offer: Offer) => void;
+  onDelete: (offer: Offer) => void;
+};
+
+export function OffersTab({ data, onAdd, onEdit, onDelete }: OffersTabProps) {
+  const [sheetOpen, setSheetOpen] = React.useState(false);
+  const [sheetMode, setSheetMode] = React.useState<'add' | 'edit'>('add');
+  const [selectedOffer, setSelectedOffer] = React.useState<Partial<Offer> | null>(null);
+  const [deleteModalOpen, setDeleteModalOpen] = React.useState(false);
+  const [offerToDelete, setOfferToDelete] = React.useState<Partial<Offer> | null>(null);
+
+  const columns = [
```

ESLint

[error] 'columns' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)